{"id":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","dependencies":[{"name":"C:\\Users\\troye\\OneDrive\\Desktop\\troyenzo\\package.json","includedInParent":true,"mtime":1706167268539},{"name":"C:\\Users\\troye\\OneDrive\\Desktop\\troyenzo\\node_modules\\smooth-scrollbar\\node_modules\\core-js\\package.json","includedInParent":true,"mtime":1706167342203},{"name":"../internals/an-object","loc":{"line":1,"column":23,"index":23},"parent":"C:\\Users\\troye\\OneDrive\\Desktop\\troyenzo\\node_modules\\smooth-scrollbar\\node_modules\\core-js\\internals\\call-with-safe-iteration-closing.js","resolved":"C:\\Users\\troye\\OneDrive\\Desktop\\troyenzo\\node_modules\\smooth-scrollbar\\node_modules\\core-js\\internals\\an-object.js"},{"name":"../internals/iterator-close","loc":{"line":2,"column":28,"index":78},"parent":"C:\\Users\\troye\\OneDrive\\Desktop\\troyenzo\\node_modules\\smooth-scrollbar\\node_modules\\core-js\\internals\\call-with-safe-iteration-closing.js","resolved":"C:\\Users\\troye\\OneDrive\\Desktop\\troyenzo\\node_modules\\smooth-scrollbar\\node_modules\\core-js\\internals\\iterator-close.js"}],"generated":{"js":"var anObject = require('../internals/an-object');\nvar iteratorClose = require('../internals/iterator-close');\n\n// call something on iterator step with safe closing on error\nmodule.exports = function (iterator, fn, value, ENTRIES) {\n  try {\n    return ENTRIES ? fn(anObject(value)[0], value[1]) : fn(value);\n  } catch (error) {\n    iteratorClose(iterator, 'throw', error);\n  }\n};\n"},"sourceMaps":{"js":{"mappings":[{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":1,"column":0},"generated":{"line":1,"column":0}},{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":2,"column":0},"generated":{"line":2,"column":0}},{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":3,"column":0},"generated":{"line":3,"column":0}},{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":4,"column":0},"generated":{"line":4,"column":0}},{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":5,"column":0},"generated":{"line":5,"column":0}},{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":6,"column":0},"generated":{"line":6,"column":0}},{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":7,"column":0},"generated":{"line":7,"column":0}},{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":8,"column":0},"generated":{"line":8,"column":0}},{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":9,"column":0},"generated":{"line":9,"column":0}},{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":10,"column":0},"generated":{"line":10,"column":0}},{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":11,"column":0},"generated":{"line":11,"column":0}},{"source":"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js","original":{"line":12,"column":0},"generated":{"line":12,"column":0}}],"sources":{"node_modules/smooth-scrollbar/node_modules/core-js/internals/call-with-safe-iteration-closing.js":"var anObject = require('../internals/an-object');\nvar iteratorClose = require('../internals/iterator-close');\n\n// call something on iterator step with safe closing on error\nmodule.exports = function (iterator, fn, value, ENTRIES) {\n  try {\n    return ENTRIES ? fn(anObject(value)[0], value[1]) : fn(value);\n  } catch (error) {\n    iteratorClose(iterator, 'throw', error);\n  }\n};\n"},"lineCount":12}},"error":null,"hash":"dfb8e7ffcf9eaf8b9da71e943dc63b52","cacheData":{"env":{}}}